tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -666,8 +686,15 @@ public function deleteFollowingFilesFromTrashbin(string $user, TableNode $table)
 		$paths = $table->getHash();
 
 		foreach ($paths as $path) {
-			$this->deleteFileFromTrashbin($user, $path["path"]);
+			$response = $this->tryToDeleteFileFromTrashbin($user, $path["path"]);
+			$numItemsDeleted = \count($response);
 
+			Assert::assertEquals(
+				1,
+				$numItemsDeleted,
+				"Expected to delete exactly one item from the trashbin but $numItemsDeleted were deleted"
+			);
+			$this->featureContext->setResponse($response[0]);
```

Comment:
Any reason for using index? In other place you have used for loop.

Reply:
Ah, needs to be similar to above one
```php
foreach ($responseArray as $response) {
	$this->featureContext->setResponse($response);
}
```

Please, address the reviews in all places where there are similar codes